<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IJEOIQ2" failures="1" tests="1" name="Default test" time="66.757" errors="0" timestamp="2021-05-09T11:28:54 IST">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="runScenario" time="65.67">
    <failure type="java.lang.AssertionError" message="expected [mobile no is not registered] but found [An OTP has been sent to your registered mobile number.]">
      <![CDATA[java.lang.AssertionError: expected [mobile no is not registered] but found [An OTP has been sent to your registered mobile number.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at stepDefinition.UnregisteredNumber.user_should_successfully_view_mobile_no_is_not_registered(UnregisteredNumber.java:50)
at âœ½.user should successfully view mobile no is not registered(src/test/resources/Feature/UnregisteredNo.feature:5)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
